api/src/main/java/no/bibsys/web/DatabaseResource.java

```diff
@@ -93,7 +98,7 @@ public Response getRegistryList(@HeaderParam(ApiKeyConstants.API_KEY_PARAM_NAME)
 
     @GET
     @Path("/{registryName}")
-    @Produces({MediaType.TEXT_HTML, MediaType.APPLICATION_JSON})
+    @Produces({MediaType.TEXT_HTML, MediaType.APPLICATION_JSON, MediaTypeRdfHelper.APPLICATION_RDF, MediaTypeRdfHelper.APPLICATION_JSON_LD, MediaTypeRdfHelper.APPLICATION_N_TRIPLES, MediaTypeRdfHelper.APPLICATION_RDF_XML, MediaTypeRdfHelper.APPLICATION_TURTLE})
```

Comment:
rename MediaTypeRdfHelper to CustomMediaType? In general the class contains only constants,  so there are a collection of MediaTypes